### ISSUE TITLE
Installer and Manjaro : systemd-resolve conflict

### ISSUE THREAD

**Issue by nam1962** [nextdns/nextdns#225]
`This device is using NextDNS with no configuration. Make sure you use the DNS-over-HTTPS endpoint shown below.` after installation.

Updated Manjaro.
```
~]$ nextdns config
log-queries false
max-ttl 0s
auto-activate true
detect-captive-portals false
setup-router true
cache-size 0
cache-max-age 0s
hardened-privacy true
timeout 5s
listen localhost:53
config xxxxxx
report-client-info true
bogus-priv true
use-hosts true
```

```
~]$ sudo ss -lnep | grep :53
udp UNCONN 0 0 0.0.0.0:53834 0.0.0.0:* users:(("avahi-daemon",pid=742,fd=13)) uid:975 ino:20972 sk:92 <->
udp UNCONN 0 0 224.0.0.251:5353 0.0.0.0:* users:(("chromium",pid=276355,fd=246)) uid:1000 ino:739168 sk:93 <->
udp UNCONN 0 0 224.0.0.251:5353 0.0.0.0:* users:(("chromium",pid=276395,fd=37)) uid:1000 ino:739612 sk:94 <->
udp UNCONN 0 0 0.0.0.0:5353 0.0.0.0:* users:(("avahi-daemon",pid=742,fd=12)) uid:975 ino:20970 sk:95 <->
udp UNCONN 0 0 0.0.0.0:5355 0.0.0.0:* users:(("systemd-resolve",pid=738,fd=11)) uid:980 ino:21860 sk:96 <->
udp UNCONN 0 0 127.0.0.1:53 0.0.0.0:* users:(("nextdns",pid=279476,fd=11)) ino:747718 sk:98 <->
udp UNCONN 0 0 127.0.0.53%lo:53 0.0.0.0:* users:(("systemd-resolve",pid=738,fd=14)) uid:980 ino:21864 sk:99 <->
tcp LISTEN 0 4096 0.0.0.0:5355 0.0.0.0:* users:(("systemd-resolve",pid=738,fd=12)) uid:980 ino:21861 sk:9b <->
tcp LISTEN 0 4096 127.0.0.1:53 0.0.0.0:* users:(("nextdns",pid=279476,fd=9)) ino:748551 sk:9d <->
tcp LISTEN 0 4096 127.0.0.53%lo:53 0.0.0.0:* users:(("systemd-resolve",pid=738,fd=15)) uid:980 ino:21865 sk:9e <-> 
```
```
~]$ cat /etc/nsswitch.conf
# Name Service Switch configuration file.
# See nsswitch.conf(5) for details.
passwd: files mymachines systemd
group: files mymachines systemd
shadow: files
publickey: files
hosts: files mymachines mdns4_minimal [NOTFOUND=return] resolve [!UNAVAIL=return] dns mdns4 myhostname
networks: files dns
protocols: files
services: files
ethers: files
rpc: files
netgroup: files
```

Seems to be solved with : 
```
~]$ systemctl stop systemd-resolved.service
~]$ systemctl disable systemd-resolved.service
Removed /etc/systemd/system/multi-user.target.wants/systemd-resolved.service.
Removed /etc/systemd/system/dbus-org.freedesktop.resolve1.service
```

But, after this, removing Nextdns with the script results in... having no DNS (need to manually insert an IP in /etc/resolv.conf )

**Comment by rs** [comment]
This distrib is having an nsswitch that uses systemd-resolved as a lib in addition to the resolv.conf file. We need to find a way to integrate better with resolved.

**Comment by bkmo** [comment]
I was unable to get the client to load without adding setup-router true. The basic workstation setup did not work.  this is with Manjaro/Arch

**Comment by dogsbody** [comment]
I feel this is the same issue with systemd-resolved as #394
